tests/fixtures/common/qualityprofile.json

```diff
@@ -0,0 +1,253 @@
+{
```

Comment:
Do we really need to statically store all of these json files within github?

Reply:
I don't think we do tbh as we only check that it's a dict or a list that is returned. 

I started doing it thinking we may assert the results of one of them at some point but then never did.